dep/g3dlite/include/G3D/Ray.h

```diff
@@ -36,7 +36,7 @@ class Ray {
     
     /** The following are for the "ray slope" optimization from
       "Fast Ray / Axis-Aligned Bounding Box Overlap Tests using Ray Slopes" 
-      by Martin Eisemann, Thorsten Grosch, Stefan M��ller and Marcus Magnor
+      by Martin Eisemann, Thorsten Grosch, Stefan M侟ller and Marcus Magnor
```

Comment:
that really doesnt look right, a chinese character instead of a german one

Reply:
That is one of the 4 warnings in AppVeyor, if I am not mistaken.

Reply:
@dxzhan : Please revert the kanji characters back to their original Latin characters.